src/components/Header/index.js

```diff
@@ -48,63 +55,87 @@ export default function Header() {
   };
 
   return (
-    <div className={classes.root}>
-      <AppBar style={{ background: "#1c1c1c" }} position="static">
-        <Toolbar>
-          <Typography variant="h6" className={classes.title}>
-            <Link className={classes.link} to="/inicio">
-              <Button color="inherit">Inicio</Button>
-            </Link>
-            <Link className={classes.link} to="/proyectos">
-              <Button color="inherit">Proyectos</Button>
-            </Link>
-            <Link className={classes.link} to="/personas">
-              <Button color="inherit">Personas</Button>
-            </Link>
-            <Link className={classes.link} to="/administradores">
-              <Button color="inherit">Administradores</Button>
-            </Link>
-          </Typography>
-          <Fragment>
-            <Tooltip title="Configuración de la cuenta">
-              <IconButton
-                aria-label="account of current user"
-                aria-controls="menu-appbar"
-                aria-haspopup="true"
-                onClick={handleMenu}
-                color="inherit"
-                data-testid="menu-perfil"
-              >
-                <AccountCircleIcon className={classes.mr5} />
-                {username}
-              </IconButton>
-            </Tooltip>
-            <Menu
-              className={classes.mt35}
-              anchorEl={anchorEl}
-              getContentAnchorEl={null}
-              open={open}
-              onClose={handleClose}
-              onClick={handleClose}
-              PaperProps={PaperProps}
-              transformOrigin={anchorElPoint}
-              anchorOrigin={anchorElPoint}
+    <Navbar>
+      <div className={classes.title}>
+        <Link className={classes.link} to="/inicio">
+          <Button>
+            <img
+              style={{ width: 13, marginRight: 9 }}
+              src={HomeIcon}
+              alt="home"
+            />
+            Inicio
+          </Button>
+        </Link>
+        <Link className={classes.link} to="/proyectos">
+          <Button>
+            <img
+              style={{ width: 13, marginRight: 9 }}
+              src={ProjectsIcon}
+              alt="home"
+            />
+            Proyectos
+          </Button>
+        </Link>
+        <Link className={classes.link} to="/personas">
+          <Button>
+            <img
+              style={{ width: 13, marginRight: 9 }}
+              src={PersonsIcon}
+              alt="home"
+            />
+            Personas
+          </Button>
+        </Link>
+        <Link className={classes.link} to="/administradores">
+          <Button>
+            <img
+              style={{ width: 13, marginRight: 9 }}
+              src={AdministratorsIcon}
+              alt="home"
+            />
+            Administradores
+          </Button>
+        </Link>
```

Comment:
I think you can encapsulate this a bit more.

You can create a component called "LinkButton" or similar and receive some props like: `style`, `icon`, `alt` and `to` or similar